test/integration/api-support/test/index.test.js

```diff
@@ -236,6 +236,11 @@ function runTests(dev = false) {
     expect(data).toEqual({ message: 'Parsed body' })
   })
 
+  it('should show friendly error for redirect', async () => {
+    const res = await fetchViaHTTP(appPort, '/api/redirect-307', null, {})
```

Comment:
It looks like this test might not have gotten updated, we need to add a separate endpoint that actually causes the error (`res.redirect(307)`) and check `stderr` for the error being present right?

Reply:
I agree. How can I add this endpoint in order to represent this error ?

Reply:
I am seeing that this has to be added in `pages/api`. Right ?

Reply:
I just added a new route to `pages/api` which force to show res.redirect friendly error @ijjk 🚀